charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-controller.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     spec:
       automountServiceAccountToken: false
       containers:
-      - name: csi-snapshot-controller
+      - name: alicloud-csi-snapshot-controller
```

Comment:
if you change container name, you may need to go through all places where uses it and change accordingly
https://github.com/gardener/gardener-extension-provider-alicloud/search?q=csi-snapshot-validation

Reply:
So, the only place I have to change it is in the [charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-validation-webhook-vpa.yaml](https://github.com/gardener/gardener-extension-provider-alicloud/blob/30d72a0273a1af2a9d44eecdbd83d603635e2832/charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-validation-webhook-vpa.yaml), correct?

All other places you linked does not link the container